src/Raven.Studio/typescript/components/pages/database/settings/connectionStrings/editForms/AzureQueueStorageConnectionString.tsx

```diff
@@ -253,7 +267,27 @@ const schema = yupObjectSchema<FormData>({
     authType: yup.string<AzureQueueStorageAuthenticationType>(),
     settings: yupObjectSchema<FormData["settings"]>({
         connectionString: yupObjectSchema<FormData["settings"]["connectionString"]>({
-            connectionStringValue: getStringRequiredSchema("connectionString"),
+            connectionStringValue: yup
+                .string()
+                .nullable()
+                .when("$authType", {
+                    is: "connectionString",
+                    then: (schema) =>
+                        schema
+                            .required()
+                            .test(
+                                "connectionString",
+                                "Please define all required fields: DefaultEndpointsProtocol, AccountName, AccountKey and QueueEndpoint",
```

Comment:
If I understand correctly from the code [here](https://github.com/ravendb/ravendb/blob/6ae5d69e9888ff917db0e1167544ca0abf549013/src/Raven.Client/Documents/Operations/ETL/Queue/AzureQueueStorageConnectionSettings.cs#L65)
then, as I wrote in the [documentation PR](https://github.com/ravendb/docs/blob/9dfdd8399fd50cfd257da7aa51b5bddad612df23/Documentation/6.1/Raven.Documentation.Pages/server/ongoing-tasks/etl/queue-etl/azure-queue.markdown?plain=1#L60),  only these are mandatory:
       * `AccountName`
       * `DefaultEndpointsProtocol`
       * `QueueEndpoint` (when using HTTP protocol)

isn't that the case ?

Reply:
If AccountKey is not present, the QueueServiceClient constructor throws an error. I checked it by testing it from the Connections string view in Stuido.

https://github.com/ravendb/ravendb/blob/8d92390c05d5841b64dae29202b375a43acbae73/src/Raven.Server/Documents/ETL/Providers/Queue/QueueBrokerConnectionHelper.cs#L130-L131

https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#connect-to-the-emulator-account-using-the-shortcut

Reply:
But you're propably right about QueueEndpoint, I will check that.

Reply:
I removed QueueEndpoint from required fields

Reply:
@kalczur 
thanks, so I will add `AccountKey` to the mandatory params in the Docs PR